docs/sources/reference/api/docker_remote_api_v1.17.md

```diff
@@ -124,6 +124,11 @@ Create a container
              "OpenStdin":false,
              "StdinOnce":false,
              "Env":null,
+             "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
yes that's better; if we're treating label values as strings, then "no" value should be an empty string